sw/device/lib/testing/lc_ctrl_testutils.c

```diff
@@ -49,15 +49,17 @@ void lc_ctrl_testutils_lc_state_log_or_die(const dif_lc_ctrl_state_t *state) {
       LOG_INFO("Life cycle state: RMA");
       break;
     default:
-      CHECK(false, "CPU is executing in locked/invalid life cycle state: %d",
+      LOG_ERROR("CPU is executing in locked/invalid life cycle state: %d",
             (uint32_t)state);
-      break;
+      return ABORTED();
```

Comment:
nit, you probably want to use
```suggestion
      return FAILED_PRECONDITION();
```
here instead, according to the guidance: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/base/internal/absl_status.h;drc=3e99db1e33bbb4988011a9bdb21ab661f666c7e9;l=142

wdyt?

Reply:
Thanks, I wasn't sure about that one, I should have read absl_status.h more carefully, thanks!